ingestion/src/metadata/ingestion/api/workflow.py

```diff
@@ -90,6 +83,16 @@ def __init__(self, config: OpenMetadataWorkflowConfig):
 
         self._retrieve_dbt_config_source_if_needed(metadata_config, service_type)
 
+        source_class = self.get(
+            self.config.source.serviceConnection.__root__.config.sourcePythonClass
+            if source_type.startswith("custom_")
```

Comment:
do we need this check? Maybe if we just have the source path that should be enough 🤔

Reply:
We don't have the `sourcePythonClass` in all the connections, only in the custom ones. The non-custom ones must keep calculating the required python class.

Reply:
oh you're right. Thanks